client/app/states/catalogs/details/details.state.js

```diff
@@ -136,7 +136,7 @@ function Controller ($stateParams, CollectionsApi, EventNotifications, ShoppingC
       targetId: vm.serviceTemplate.id,
       targetType: 'service_template'
     }
-    const url = `${vm.dialogUrl}/${vm.serviceTemplate.id}`
+    const url = `${vm.dialogUrl}/${vm.dialogId}`
```

Comment:
Ah, this won't work. You'll either have to use the id from one of the previous lines (`idList.dialogId`) or use `vm.parsedDialogs.id`.